### PR TITLE
fix(condo): DOMA-4258 fixed cron according to UTC

### PR DESCRIPTION
**Pull request by akarjakin**

### CODE REVIEW

**Review by AntonAL — APPROVED** [review]

File: apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js [hunk 1]
```diff
@@ -2,10 +2,12 @@ const { createCronTask } = require('@condo/keystone/tasks')
 
 const { sendBillingReceiptNotifications } = require('./helpers')
 
-const WORK_DAYS_FROM = 10
-const WORK_DAYS_TO = 15
-const WEEKENDS_FROM = 12
-const WEEKENDS_TO = 15
+/** PODS work according to UTC, so time in cron-tasks should also be according to UTC too. */
+const UTC_TO_MSC = -3
```
Comment (pahaz) [inline apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js:6]: you can not change the scheduling time like so!

you must remove the old one by `removeCronTask` and add the new one by `createCronTask`

Comment (akarjakin) [reply apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js:6]: fixed

File: apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
-const sendBillingReceiptNotificationsTaskWeekends = createCronTask('sendBillingReceiptNotificationsTaskWeekends', `17 ${WEEKENDS_FROM}-${WEEKENDS_TO} * * 0,6`, sendBillingReceiptNotifications)
+const UTC_WORK_DAYS_FROM = MSC_WORK_DAYS_FROM + UTC_TO_MSC
+const UTC_WORK_DAYS_TO = MSC_WORK_DAYS_TO + UTC_TO_MSC
+const UTC_WEEKENDS_FROM = MSC_WEEKENDS_FROM + UTC_TO_MSC
+const UTC_WEEKENDS_TO = MSC_WEEKENDS_TO + UTC_TO_MSC
+
+removeCronTask('sendBillingReceiptNotificationsTaskWorkDays', `17 ${MSC_WORK_DAYS_FROM}-${MSC_WORK_DAYS_TO} * * 1-5`)
+removeCronTask('sendBillingReceiptNotificationsTaskWeekends', `17 ${MSC_WEEKENDS_FROM}-${MSC_WEEKENDS_TO} * * 0,6`)
+
+const sendBillingReceiptNotificationsTaskWorkDays = createCronTask('sendBillingReceiptNotificationsTaskWorkDays', `17 ${UTC_WORK_DAYS_FROM}-${UTC_WORK_DAYS_TO} * * 1-5`, sendBillingReceiptNotifications)
```
Comment (pahaz) [inline apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js:20]: Could you please HARDCODE the value. it's hard to read the real value! You don't need to reuse this consts >_<

Comment (pahaz) [reply apps/condo/domains/resident/tasks/sendBillingReceiptNotifications.task.js:20]: It's hard to check it

**Review by pahaz — APPROVED** [review]

**Pull request merged**